scripts/reset_default.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
Please add a module docstring that 
- explains what this script is goof for, 
- how/when/where it is automatically triggered, and 
- provides an example instructing users how they can run the script locally.